single_cell_condensed_sdrf.sh

```diff
@@ -230,18 +230,7 @@ if [ "$replicates" = true ]; then
   technicalReplicatesOption="--mergeTechReplicates"
 fi
 
-zoomaOption="-z"
-if [ -n "$skipZooma" ]; then
-  zoomaOption=""
-fi
-
-# Pass the same exclusions file to both downstream scrips
-exclusions=
-
-if [ -e "$zoomaExclusions" ]; then
-exclusions=" -x $zoomaExclusions"
-fi
-condense_sdrf.pl -e $expId -fi $idfFile $technicalReplicatesOption -sc $zoomaOption -o $outputDir $exclusions
+condense_sdrf.pl -e $expId -fi $idfFile $technicalReplicatesOption -sc -o $outputDir $exclusions
```

Comment:
I'm confused here, this seems to me as it zooma would be running with this condensed SDRF run, I though that you wanted to do a single zooma run only at the end.

Reply:
or does it skip zooma by default if not given -z

Reply:
Yes it does, see https://github.com/ebi-gene-expression-group/experiment_metadata/blob/233e6f87d9bb8a013ff1fe5bce788ae8c7cd5d5d/condense_sdrf.pl#L61